src/libs/PolicyUtils.ts

```diff
@@ -538,6 +539,49 @@ function canUseProvincialTaxNetSuite(subsidiaryCountry?: string) {
     return subsidiaryCountry === '_canada';
 }
 
+function getNetSuiteReimbursableAccountOptions(policy: Policy | undefined, selectedBankAccountId: string | undefined): SelectorType[] {
+    const payableAccounts = policy?.connections?.netsuite.options.data.payableList ?? [];
+    let accountsToConsider = (payableAccounts ?? []).filter(({type}) => type === '_bank');
+    accountsToConsider = accountsToConsider.concat((payableAccounts ?? []).filter(({type}) => type === '_creditCard'));
```

Comment:
Couldn't we combine the condition in filter `['_bank', '_creditCard'].includes(type)` and then if sort if we really want `_bank` to be first?